io.github.giantpinkrobots.varia.json

```diff
@@ -0,0 +1,155 @@
+{
+    "app-id" : "io.github.giantpinkrobots.varia",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "varia",
+    "_justifications": "This program utilizes aria2c, and aria2c requires the networking permission and the xdg-download directory in order to download files.",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=xdg-download"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+	{
+	    "name": "python3-aria2p",
```

Comment:
if these are auto generated (look like it) they should be in a separate json, include here, to allow regeneration of the sources.

Reply:
Yes, the aria2p module was auto generated. Originally it was a separate json, I could just upload it here and include it. Or do you mean link to my original repository to fetch that json from there?